components/Searchbar/index.js

```diff
@@ -58,7 +59,7 @@ export default function Searchbar({
     setShowSuggestions(false);
     onClickEvent(inputValue);
   }
-
+  console.log(inputValue);
```

Comment:
I prefer to remove `console.log()` before a deploy, because this is actually only information that is relevant during development.